com.github.giacomogroppi.writernote-qt.json

```diff
@@ -0,0 +1,88 @@
+{
+  "app-id": "com.github.giacomogroppi.writernote-qt",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "6.2",
+  "sdk": "org.kde.Sdk",
+  "command": "writernote",
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--share=ipc",
+    "--filesystem=xdg-run/pipewire-0",
+    "--filesystem=host",
+    "--talk-name=org.kde.StatusNotifierWatcher",
+    "--talk-name=org.freedesktop.Flatpak",
+    "--talk-name=org.freedesktop.Notifications",
+    "--talk-name=org.a11y.Bus",
+    "--own-name=org.kde.StatusNotifierItem-2-2",
+    "--system-talk-name=org.freedesktop.Avahi"
+  ],
+  "add-extensions": {
+  },
+  "cleanup": [
+    "/lib/pkgconfig",
+    "/share/man",
+    "*.la"
+  ],
+  "modules": [
+    {
+      "name": "Libzip",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DCMAKE_INSTALL_LIBDIR=/app/lib",
+        "-DCMAKE_INSTALL_INCLUDEDIR=7app/include"
+      ],
+      "cleanup": [
+        "/bin"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/nih-at/libzip.git",
+          "tag": "v1.8.0",
+          "commit": "26ba5523db09213f532821875542dba7afa04b65"
+        }
+      ]
+    },
+    {
+      "name": "Poppler",
+      "buildsystem": "cmake-ninja",
+      "config-opts":[
+        "-DCMAKE_INSTALL_LIBDIR=/app/lib",
+        "-DCMAKE_INSTALL_INCLUDEDIR=/app/include",
+        "-DENABLE_BOOST=OFF",
```

Comment:
I'll assume you made sure the "Splash" backend wasn't used.

Reply:
Yes, I made sure that it wasn’t used